assets.go

```diff
@@ -2,4 +2,8 @@
 
 package main
 
+import "net/http"
+
 const production = false
```

Comment:
Ahh, I see. I agree with your reasoning then. At first glance it seemed incorrect but I would much prefer to see `if production` rather than `if !dev`.